internal/testserver/server.go

```diff
@@ -131,6 +132,10 @@ func NewTestServerWithConfigAndDatastore(require *require.Assertions,
 						Name:       "servicespecific",
 						Middleware: servicespecific.UnaryServerInterceptor,
 					},
+					{
```

Comment:
This is suspicious. Why do we have this custom server setup here? It seems it's only used by the steelthread tests. Why can't those tests use a default server, why do they need a special arrangement of middleware?

Reply:
Because we don't need all the other middleware running and all it does it increase the test time to have them running

Reply:
How could middleware add so much test time that we decide to not include them?